hack/kokoro-test.sh

```diff
@@ -25,3 +25,4 @@ echo "Running integration tests..."
 # Generate the integration test yaml and pass it to cloud build via stdin.
 python ftl/ftl_node_integration_tests_yaml.py | gcloud container builds submit --config /dev/fd/0 .
 gcloud container builds submit --config ftl/ftl_python_integration_tests.yaml .
+gcloud container builds submit --config ftl/ftl_php_integration_tests.yaml .
```

Comment:
can you change this to a yaml test generator like the node tests?  That way it'll be easier to extend in the future and we can dedupe some of test logic